CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.8)
 project(bn256 VERSION 1.0.0)
```

Comment:
I think it is worth updating the version on changing from C++-17 to C++20. Maybe `1.1.0`. Would be good to create a `release/1.0` branch with current code. I could see projects that are using C++-17 liking to have a C++17 version of the library available. 

Note the README.md also needs to be updated with the correct version.

Reply:
done.